src/System.IO.Compression.Brotli/src/System.IO.Compression.Brotli.csproj

```diff
@@ -9,8 +9,8 @@
   <!-- Default configurations to help VS understand the options -->
   <!-- Platform agnostic files -->
   <ItemGroup>
-    <Compile Include="Interop\Interop.Brotli.Encoder.cs" />
-    <Compile Include="Interop\Interop.Brotli.Decoder.cs" />
+    <Compile Include="$(CommonPath)\Interop\Interop.Brotli.Encoder.cs" />
+    <Compile Include="$(CommonPath)\Interop\Interop.Brotli.Decoder.cs" />
```

Comment:
I'd prefer to see all of the interop signatures for Brotli in a single file, especially if it's going to just be sitting in the root Interop folder.

Reply:
Ok, I'll move everything into a single "Interop.Brotli.cs" file.